tests/next_tests/unit_tests/iterator_tests/transforms_tests/test_inline_lambdas.py

```diff
@@ -56,3 +56,24 @@ def test(name, opcount_preserving, testee, expected):
 
     inlined = InlineLambdas.apply(testee, opcount_preserving=opcount_preserving)
     assert inlined == expected
+
+
+def test_inline_lambda_args():
+    testee = im.let("reduce_step", im.lambda_("x", "y")(im.plus("x", "y")))(
+        im.lambda_("a")(
+            im.call("reduce_step")(im.call("reduce_step")(im.call("reduce_step")("a", 1), 2), 3)
+        )
+    )
+    expected = im.lambda_("a")(
+        im.call(im.lambda_("x", "y")(im.plus("x", "y")))(
+            im.call(im.lambda_("x", "y")(im.plus("x", "y")))(
+                im.call(im.lambda_("x", "y")(im.plus("x", "y")))("a", 1), 2
+            ),
+            3,
+        )
+    )
+    inlined = InlineLambdas.apply(
+        testee,
+        force_inline_lambda_args=True,
```

Comment:
```suggestion
        opcount_preserving=True,
        force_inline_lambda_args=True,
```
Otherwise everything is inlined unconditionally and the flag has no effect.